_rules/video-alternative-for-visual-c5a4ea.md

```diff
@@ -138,7 +138,9 @@ A video element with a track element that contains descriptions.
 
 #### Passed Example 4
 
-A video element that describes some of the text on the same page. The text on the page labels the video as an alternative.
+The video element describes some of the text on the same page. The text identifies the video as an alternative.
```

Comment:
```suggestion
This video element describes some of the text on the same page. The text identifies the video as an alternative.
```
We normally use demonstrative article in test cases description: https://act-rules.github.io/pages/design/rule-design/#test-cases